extensions/amp-analytics/0.1/test/vendor-requests.json

```diff
@@ -5,6 +5,12 @@
     "pageview": "https://$log$domain/hit.xiti?s=$site&ts=_timestamp_&r=_screen_width_x_screen_height_x_screen_color_depth_&re=_available_screen_width_x_available_screen_height_&p=_title_&s2=$level2&ref=_document_referrer_",
     "click": "https://$log$domain/hit.xiti?s=$site&ts=_timestamp_&r=_screen_width_x_screen_height_x_screen_color_depth_&re=_available_screen_width_x_available_screen_height_&pclick=_title_&s2click=$level2&p=$label&s2=$level2Click&type=click&click=$type&ref=_document_referrer_"
   },
+  "burt": {
+    "host": "//$trackingKey.c.richmetrics.com/",
+    "base": "//$trackingKey.c.richmetrics.com/log?e=$trackingKey&pi=$trackingKey|_page_view_id_|_canonical_path_|_client_id_&v=amp&t=_timestamp_&sn=1&",
```

Comment:
s/log/imglog

here and below. Tests fail because of this and possibly other reasons.